examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -234,7 +260,7 @@ CHIP_ERROR PairingCommand::EnableNetwork()
     }
     else
     {
-        networkId = ByteSpan(kTemporaryThreadNetworkId, sizeof(kTemporaryThreadNetworkId));
+        networkId = GetThreadNetworkId();
```

Comment:
Can we add a check here to check that `networkId` is not empty and returns an error otherwise ? Otherwise the code will continue as if.

Reply:
Addressed all reviews but it got autoclosed during force push, had to open new one because Reopen button is disabled.

https://github.com/project-chip/connectedhomeip/pull/8245